modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/AbstractCacheDumpTest.java

```diff
@@ -351,8 +360,8 @@ else if (data.config().getName().equals(CACHE_1)) {
                         throw new IgniteException("Unknown cache");
                 });
 
-                for (boolean found : cachesFound)
-                    assertTrue(found);
+                for (int i = 0; i < cachesFound.length; i++)
```

Comment:
cachesFound sorted? Do we have guarantee that order of caches in cachesFound will be the same as we expected?

Reply:
Updated the logic to compare 2 Sets.